cmd/examples/test-dataplane/test-dataplane.go

```diff
@@ -600,7 +604,19 @@ func connectPods(k8s *kubernetes.Clientset, podName1, podName2, namespace1, name
 	if err := listInterfaces(ns2); err != nil {
 		logrus.Errorf("Failed to list interfaces of %s/%swith error: %+v", namespace2, podName2, err)
 	}
-	return nil
+	connection := dataplaneapi.Connection{
+		LocalSource: &common.LocalMechanism{
+			Type:       common.LocalMechanismType_KERNEL_INTERFACE,
+			Parameters: map[string]string{"clientlocalmechanism": interfaces[0]},
+		},
+		Destination: &dataplaneapi.Connection_Local{
+			Local: &common.LocalMechanism{
+				Type:       common.LocalMechanismType_KERNEL_INTERFACE,
+				Parameters: map[string]string{"nselocalmechanism": interfaces[1]},
```

Comment:
My general comment here mirrors what's above.  If these are interface names, lets call them that.